pkg/controller/operators/olm/requirements_test.go

```diff
@@ -485,87 +485,7 @@ func TestRequirementAndPermissionStatus(t *testing.T) {
 			},
 			met: false,
 			expectedRequirementStatuses: map[gvkn]v1alpha1.RequirementStatus{
-				{"apiextensions.k8s.io", "v1beta1", "CustomResourceDefinition", "c1.g1"}: {
-					Group:   "apiextensions.k8s.io",
-					Version: "v1beta1",
-					Kind:    "CustomResourceDefinition",
-					Name:    "c1.g1",
-					Status:  v1alpha1.RequirementStatusReasonNotAvailable,
-				},
-				{"operators.coreos.com", "v1alpha1", "ClusterServiceVersion", "csv1"}: {
-					Group:   "operators.coreos.com",
-					Version: "v1alpha1",
-					Kind:    "ClusterServiceVersion",
-					Name:    "csv1",
-					Status:  v1alpha1.RequirementStatusReasonPresent,
-				},
-			},
-			expectedError: nil,
-		},
-		{
-			description: "RequirementNotMet/NamesConflictedCRD",
```

Comment:
It seems like we'd want a test to make sure that name conflicts don't occur. Is there another test that protects against this bug?